NEWS.md

```diff
@@ -9,8 +9,11 @@ The new features include:
 * Almost all arguments of the geocoding functions are vectorised (the exceptions being `output`, `key` and `no_record`), so it is possible to serially (reverse) geocode lists of placenames or coordinates. The geocoding functions show a progress indicator when more than one `placename` or `latitude`/`longitude` pair is provided.
 * The forward geocoding functions now support multiple `countrycode`s in accordance with the OpenCage API (#44). The `countrycode`s can now be provided in upper or lower case (#47).
 * A helper function `oc_bbox()` now makes it easier to create (lists of) bounding boxes from vectors, bbox objects and data frames. 
+* The forward geocoding functions now support the new `proximity` argument (#60). The results of the geocoding request will be biased towards that location, see [this post](https://blog.opencagedata.com/post/new-optional-parameter-proximity) on the OpenCage blog.
```

Comment:
Grammar: The results of the geocoding request will be biased towards that location, see [this post](https://blog.opencagedata.com/post/new-optional-parameter-proximity) on the OpenCage blog.

Should be  The results of the geocoding request will be biased towards that location. See [this post](https://blog.opencagedata.com/post/new-optional-parameter-proximity) on the OpenCage blog.